plugins/wasmedge_opencvmini/opencvmini_func.cpp

```diff
@@ -123,14 +123,16 @@ Expect<void> WasmEdgeOpenCVMiniImencode::body(
 
 Expect<uint32_t>
 WasmEdgeOpenCVMiniNormalize::body(const Runtime::CallingFrame &,
-                                  uint32_t SrcMatKey) {
+                                  uint32_t SrcMatKey, uint32_t Alpha,
+                                  uint32_t Beta, uint32_t NormType) {
   auto Src = Env.getMat(SrcMatKey);
   if (!Src) {
     return Unexpect(ErrCode::Value::HostFuncError);
   }
   cv::Mat Dst;
-  // convert each elements `v` of `Src` to `(1/255) * v + 0`
-  Src->convertTo(Dst, CV_32F, 1. / 255., 0.);
+  // -1 below is the dtype: We want the output array to have same type as input
+  // array
+  cv::normalize(*Src, Dst, Alpha, Beta, NormType, -1, cv::noArray());
```

Comment:
hard coded arguments `-1` and `cv::noArray()`

Reply:
Updated, Ive split normalize into two variants, 
`normalize`, expecting all arguments, including a Key to a mask,
And `normalize_no_mask`, using `cv::noArray()` for developer convenience.

An alternative is to expose a function to create an empty array in the environment, returning a key of that empty array, 
Then let the developer use that and only expose the normalize function. 

@dannypsnl 
Let me know if you prefer that, or if you are happy with the two function approach.

Reply:
The first one is enough, we can wrap it up in final library to get convience